sql/core/src/test/resources/sql-tests/inputs/group-by-filter.sql

```diff
@@ -1,4 +1,7 @@
--- Test filter clause for aggregate expression.
+-- Test filter clause for aggregate expression with codegen on and off.
+--CONFIG_DIM1 spark.sql.codegen.wholeStage=true
+--CONFIG_DIM1 spark.sql.codegen.wholeStage=false,spark.sql.codegen.factoryMode=CODEGEN_ONLY
```

Comment:
I can't understand this dimension config.
When `spark.sql.codegen.wholeStage=false` and `spark.sql.codegen.factoryMode`.
It means each operator will be compiled into single java and cannot fallback to interpreted ?

Reply:
That means the plan interpreter is  used with the expression codegen (e.g., `GenerateUnsafeProjection`) enabled.

Reply:
Thanks. I learned it.